### PR TITLE
Changed delimeter character

### PR DESCRIPTION
**Pull request by thomasfanell**
also some readability changes.

### CODE REVIEW

File: source/io/CsvReader.ooc [hunk 1]
```diff
@@ -33,7 +35,7 @@ CsvReader: class extends Iterator<VectorList<Text>> {
 		readCharacter: Char
 		for (i in 0 .. rowDataLength) {
 			textBuilder := TextBuilder new()
-			while (i < rowDataLength && ((readCharacter = rowData[i]) != ';')) {
+			while (i < rowDataLength && ((readCharacter = rowData[i]) != ',')) {
```
Comment (davidhesselbom) [inline source/io/CsvReader.ooc]: Maybe it would be a good idea to use static variables for the delimiter chars?

**Comment by thomasfanell** [comment]
Moved the delimiter to a static property in CsvReader. I don't know if this is the best approach, but I don't really like the idea of having two static variables, one for each class.

File: source/io/CsvWriter.ooc [hunk 2]
```diff
@@ -15,9 +15,17 @@ CsvWriter: class {
 	}
 	write: func (row: VectorList<Text>) {
 		for (i in 0 .. row count) {
-			this _fileWriter file write(row[i] toString())
+			value := TextBuilder new (row[i])
```
Comment (davidhesselbom) [inline source/io/CsvWriter.ooc]: `TextBuilder new(row[i])`. magic doesn't catch this... odd.

File: source/io/CsvWriter.ooc [hunk 3]
```diff
@@ -15,9 +15,17 @@ CsvWriter: class {
...
+			for (k in 0 .. row[i] count)
+				if (this _isWhitespace(row[i][k])) {
+					value prepend('\"')
+					value append('\"')
+					break
+				}
 			if (i < row count - 1)
-				this _fileWriter write(";")
+				value append(CsvReader delimiter)
+			this _fileWriter file write(value toString())
```
Comment (davidhesselbom) [inline source/io/CsvWriter.ooc]: `this _fileWriter write` should work.

Comment (thomasfanell) [inline source/io/CsvWriter.ooc]: ```FileWriter write()``` takes either ```(Char*, SizeT)```, or ```(Char)```. I want to write the entire string without having to provide length. Is this preferable: ```this _fileWriter write(value raw, value count as SizeT)```? ```SizeT``` is as you know platform dependant.

File: source/io/CsvWriter.ooc (hunk 2 above)
Comment (thomasfanell) [inline source/io/CsvWriter.ooc]: Should it?

File: source/io/CsvWriter.ooc (hunk 3 above)
Comment (davidhesselbom) [inline source/io/CsvWriter.ooc]: Mm. Maybe we should add a function in FileWriter that takes a `String`?

Comment (thomasfanell) [inline source/io/CsvWriter.ooc]: Do you mean that I should do that in this PR?

Comment (davidhesselbom) [inline source/io/CsvWriter.ooc]: I suppose it can wait.

**Pull request merged**